src/Raven.Server/Properties/Settings/settings.docker.posix.json

```diff
@@ -0,0 +1,5 @@
+﻿{
+    "Setup.Mode": "Initial",
+    "DataDir": "RavenData",
```

Comment:
data dir should be `/databases`. It's exposed as a volume so that use can mount it to another container and should be set in env variable RAVEN_DataDir (old DATA_DIR must be renamed)

Reply:
also tcp port must be set to 38888 somehow, since settings are overriden on setup, not sure how to achieve that other way that through CLI_ARGS or somehow merging settings from before setup with the one after. Thoughts?